### ISSUE TITLE
Unable to overwrite default key bindings

### ISSUE THREAD

**Issue by kidmose** [mitmproxy/mitmproxy#3237]
##### Steps to reproduce the problem:

1. Enter the following into `~/.mitmproxy/keys.yaml`, expecting the
  effect to be that `n` gets bound to `console.nav.down`:

        -
          key: n
          cmd: console.nav.down

2. Start `mitmproxy`
3. Press `n`
4. Observe that `n` is still bound to `console.command view.create get https://example.com/`,
  as per https://github.com/mitmproxy/mitmproxy/blob/master/mitmproxy/tools/console/defaultkeys.py#L61

##### Any other comments? What have you tried so far?

This seems to be a conflict between the implemented behaviour and the
expected behaviour (At least my expectations).

My motivation for this possibly unanticipated key binding is that I'd
like try and make bindings similar to the defaults of emacs dired
mode, which would make mitmproxy easier to use for me.

With a few hints on where to fix this and some guidance, I'd be happy to try and do it my self.

##### System information

	Mitmproxy: 4.0.3 (+9, commit d08c9cf)
	Python:    3.6.5
	OpenSSL:   OpenSSL 1.1.0h  27 Mar 2018
	Platform:  Linux-4.8.0-53-generic-x86_64-with-debian-stretch-sid

**Comment by kajojify** [comment]
@kidmose Hello! Thanks for the report. 
Yes, it seems, that key bindings from `defaultkeys.py` have priority over the key bindings from `~/.mitmproxy/keys.yaml`.
I checked your key binding. It worked for all contexts except `flowlist`, since it is set by `defaultkeys.py`. The temporary solution is to set `n` key binding like:
```
 -
   key: n
   ctx: ["global", "flowlist"]
   cmd: console.nav.down
```
explicitly changing key binding for the context(s), which is set by `defaultkeys.py`.

**Comment by kidmose** [comment]
Thanks for the temporary solution. I can confirm that it works.

**Comment by mhils** [comment]
I think user keybindings should overwrite defaultkeys, right?

**Comment by Kriechi** [comment]
fixed by #3340
